brands/README.md

```diff
@@ -1,16 +1,18 @@
 # Brands
 
-After creating an account and having access to the dashboard, the next step is to create a test **Brand** so you can create locations and add them to a Program in order to start tracking transactions.
+After creating an account and having access to the dashboard, the next step is to create a test **Brand** (Merchant) so you can create locations and add them to a Program in order to start tracking transactions. 
 
-The **Brand** (Merchant) object is used to aggregate locations and keep the status of the Brand consent. One of the requirements, when creating a Brand and before adding locations, is to obtain Brand consent from an authorized personnel (Brand User) and to provide the Brand with access to view the transactional data that is being shared.
+>Once you create a **Brand**, it cannot be deleted.
```

Comment:
We don't need the `>`, it can be just a simple sentence. I'd move this sentence below the next paragraph.